python/tank/deploy/tank_commands/validate_config.py

```diff
@@ -145,54 +145,54 @@ def _validate_bundle(log, tk, name, settings, descriptor):
     
     
     manifest = descriptor.get_configuration_schema()
-    
+
     for s in settings.keys():
         if s not in manifest.keys():
             log.info("  WARNING - Parameter not needed: %s" % s)
-        
-        else: 
-            default = manifest[s].get("default_value")
-            
-            value = settings[s]
-            try:
-                validation.validate_single_setting(name, tk, manifest, s, value)
-            except TankError, e:
-                log.info("  ERROR - Parameter %s - Invalid value: %s" % (s,e))
-            else:
-                # validation is ok
-                if default is None:
-                    # no default value
-                    # don't report this
-                    pass
-                    #log.info("  Parameter %s - OK [no default value specified in manifest]" % s)
-                
-                elif manifest[s].get("type") == "hook" and value == "default":
-                    # don't display when default values are used.
-                    pass
-                    #log.info("  Parameter %s - OK [using hook 'default']" % s)
-                
-                elif default == value:
-                    pass
-                    # don't display when default values are used.
-                    #log.info("  Parameter %s - OK [using default value]" % s)
-                
-                else:
-                    log.info("  Parameter %s - OK [using non-default value]" % s)
-                    log.info("    |---> Current: %s" % value)
-                    log.info("    \---> Default: %s" % default)
-                    
-                # remember templates
-                if manifest[s].get("type") == "template":
-                    g_templates.add(value)
-                if manifest[s].get("type") == "hook":
-                    g_hooks.add(value)
 
-                     
-    for r in manifest.keys():
-        if r not in settings.keys():
-            log.error("Required parameter missing: %s" % r)
+    for s in manifest.keys():
+
+        default = bundle.resolve_default_value(manifest.get(s), engine_name=engine_name)
```

Comment:
now uses `manifest[s]` instead.

Yep, looks like you saw where the engine name was coming from in a later comment. It's just the paren't bundle in the config. I've since removed the `app_name` argument since it wasn't used. I'd included thinking it would be needed, but it isn't.